tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv

```diff
@@ -9,3 +9,4 @@
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageAccountManagementPolicyCommand","Remove-AzureRmStorageAccountManagementPolicy","1","8600","Cmdlet 'Remove-AzureRmStorageAccountManagementPolicy' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageContainerCommand","Remove-AzureRmStorageContainer","1","8600","Cmdlet 'Remove-AzureRmStorageContainer' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageAccountManagementPolicyCommand","Remove-AzureRmStorageAccountManagementPolicy","1","8600","Cmdlet 'Remove-AzureRmStorageAccountManagementPolicy' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.GetAzureStorageAccountCommand","Get-AzureRmStorageAccount","1","8410","Parameter IncludeGeoReplicationStats of cmdlet Get-AzureRmStorageAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
Let's quickly discuss in the meeting whether this parameter should be plural - I did not notice in the cmdlet review, which was my fault.

Reply:
IncludeGeoReplicationStats, means Include GeoReplication Status. This is just the name aligned with SDK and rest api, suggest not change it.

Reply:
As we discussed in the meeting ,this should be good for switch parameter, and don't need change it.